dnf-behave-tests/features/plugins-core/versionlock-lock.feature

```diff
@@ -96,6 +96,20 @@ Scenario: I can remove the installed locked version of the package
         | remove        | wget-0:1.19.5-5.fc29.x86_64           |
 
 
+@bz1780370
+Scenario: I can remove installed package when other version is locked
+  Given I successfully execute dnf with args "install wget-1.19.5"
+    And I create file "/etc/dnf/plugins/versionlock.list" with
+    """
+    wget-0:1.19.6-5.fc29.*
```

Comment:
How about making the version more visibly different? Squinting required to discern this one...

Reply:
Done.